lib/cryptoexchange/exchanges/txbit/services/market.rb

```diff
@@ -33,7 +33,7 @@ def adapt(output, market_pair)
           ticker.ask       = NumericHelper.to_d(market['Ask'])
           ticker.bid       = NumericHelper.to_d(market['Bid'])
           ticker.volume    = NumericHelper.to_d(market['BaseVolume'])
```

Comment:
Volume should be in `base currency` instead of `target currency`, can replace L35 with the line below to convert the volume to base currency:-
`ticker.volume    = NumericHelper.divide(NumericHelper.to_d(market['BaseVolume']), ticker.last)`

Thanks.

Reply:
The volume for `base currency` is also in our feed. Instead of the conversion I used `Volume` as source for the data:

```json
{
  "success": true,
  "message": "",
  "result": {
    "MarketName": "XLR/BTC",
    "High": 0.00018499,
    "Low": 0.000142,
    "Volume": 4773.936,
    "Last": 0.00018499,
    "BaseVolume": 0.83124521,
    "TimeStamp": "2019-03-04T23:24:36.283Z",
    "Bid": 0.00014,
    "Ask": 0.00017799,
    "OpenBuyOrders": 15,
    "OpenSellOrders": 78,
    "PrevDay": 0.000178,
    "Created": "2019-01-26T11:04:35.473Z",
    "DisplayMarketName": "Solaris - Bitcoin"
  }
}
```

Thanks again @ElvisLYC